test/e2e/map.test.js

```diff
@@ -74,4 +74,13 @@ describe('vl-map', async () => {
     const map = await vlMapPage.getMap();
     await assert.eventually.equal(map.getScale(), '50 km');
   });
+
+  it('als gebruiker kan ik een kaart fullscreen aan- en uitzetten', async () => {
+    const map = await vlMapPage.getMap();
+    const mapWithFullscreenAllowed = await vlMapPage.getMapWithFullscreenAllowed();
+    await assert.eventually.isFalse(map.isFullscreenAllowed());
+    await assert.eventually.isTrue(mapWithFullscreenAllowed.isFullscreenAllowed());
+    await mapWithFullscreenAllowed.toggleFullscreen();
+    await mapWithFullscreenAllowed.toggleFullscreen();
```

Comment:
checken dat er ook effectief iets gebeurt?

Reply:
Ik zal eens kijken hoe ik daar nog een extra check voor kan voorzien, met width en height kon het niet gedetecteerd worden. Misschien geeft `document.fullscreen` een correcte boolean, dan kan ik nog een assert toevoegen.

Reply:
er zou toch moeten een of andere div of zo een andere breedte en of hoogte gekregen hebben zou ik denken

Reply:
Ja, maar dan ben je niet zeker dat het alleen wat groter is of dat het fullscreen is.

Reply:
zien dat het verschil tussen de size van uwen div en de size van uw window < X is?

Reply:
Window beperkt zich tot je browser. Fullscreen neemt de volledige breedte van je scherm in, niet alleen dat van je browser waardoor het onvoldoende (zelf niet correct) is om daar op te controleren. Ondertussen werden de testen aangevuld met een assert.